Login info for README.md

Houkime opened the pull request:
Adding info on how to connect and login into your Jenkins installation as an admin.
Took a full Jenkins noob (me) awhile to deduce, so want to save someone else's time.

Commit:
note about sudo/docker req when running images

doesn't work otherwise.

Commit:
Merge pull request #2 from Houkime/sudoreqnote

note about sudo/docker req when running images

Commit:
Revert "note about sudo/docker req when running images"

Commit:
Merge pull request #4 from Houkime/revert-2-sudoreqnote

Revert "note about sudo/docker req when running images"

Commit:
Add default login info and port note.

It took me a bit to deduce this since I'm unfamiliar with Jenkins, so let's save another contributors' time.

Commit:
Update README.md

oleg-nenashev commented on README.md:
```diff
@@ -43,6 +43,14 @@ If you use Docker for Mac, use `-Dio.jenkins.dev.host` and additional `socat` im
 
+Connect to Jenkins and login:
+
+Open 127.0.0.1:8080 (by default) in your browser while the server is running.  
```
It's not a default address. The default address is actually `DOCKER_HOST:8080` though it becomes a bit more complicated in Docker for Windows/Mac

Houkime replied:
Ok, it seems like it should be a minisection on how to detect/set your factual server address. Will dive into it a bit. Worth doing because this obviously won't be usable otherwise. And plus there is a Dockerfile which in fact requests but ignores your DEV_HOST input.

Commit:
Add IP address detection

Trying to make finding your server IP easier with some automation.
Cause it's not really cool to make users wrap their heads around Docker network mess just to login.

Commit:
"reachable" correction

There's a thing in Jenkins that graphical launcher doesn't really work externally by default.

https://stackoverflow.com/questions/26570128/jenkins-server-is-not-accessible-by-host-name-ip-address

Houkime commented:
Ok, tried another approach. Instead of writing a mundane instruction in docs, tried to autodetect possible server ip addresses on Jenkins start. Right from inside the container. However, the most agnostic approach through external server IP check does not really work because Jenkins doesn't work externally by default (it has internet connection but you cannot reach it's GUI from outside localhost or using WAN IP).

Commit:
Update README.md

Commit:
add docker host IP detection

more possible IPs

oleg-nenashev reviewed (approved):
works for me
